package.json

```diff
@@ -1,31 +1,8 @@
 {
   "name": "@mojaloop/event-stream-processor",
-  "version": "v8.1.0",
+  "version": "8.3.3-snapshot",
```

Comment:
Why are you creating a snapshot?

Reply:
Because I've seen the last 3 releases are snapshots. Shall I change it to 8.3.3 and create a normal release?

Reply:
@rmothilal @ggrg  We are still reworking and changed the logger technology before. Also no tests or proper error handling yet. So we shall keep it still as a snapshot, I believe.